spec/services/meeting_starter_spec.rb

```diff
@@ -60,13 +62,40 @@
     it 'broadcasts to ActionCable that the meeting has started' do
       allow_any_instance_of(BigBlueButtonApi)
         .to receive(:start_meeting)
-        .and_return(true)
+        .with(room:, options:, presentation_url:)
+        .and_return(meeting_starter_response)
 
       expect(ActionCable.server)
         .to receive(:broadcast)
         .with("#{room.friendly_id}_rooms_channel", 'started')
 
       service.call
     end
+
+    it 'updates the last session date when a meeting is started' do
+      allow_any_instance_of(BigBlueButtonApi)
+        .to receive(:start_meeting)
+        .with(room:, options:, presentation_url:)
+        .and_return(meeting_starter_response)
+
+      service.call
+
+      expect(room.last_session).to eql(DateTime.strptime(1_389_464_535_956.to_s, '%Q').utc)
```

Comment:
I wouldn't hardcode the startTime here. 

`expect(room.last_session).to eql(DateTime.strptime(meeting_starter_response[:createTime].to_s, '%Q').utc)`